cmd/clusterctl/README.md

```diff
@@ -44,7 +44,7 @@ https://github.com/kubernetes-sigs/cluster-api/issues/158 and https://github.com
    ./clusterctl create cluster --provider <provider> -c cluster.yaml -m machines.yaml -p provider-components.yaml -a addons.yaml
    ```
 
-To choose a specific minikube driver, please use the `--vm-driver` command line parameter. For example to use the kvm2 driver with clusterctl you would add `--vm-driver kvm2`
+To choose a specific minikube driver, please use the --bootstrap-flags vm-driver=xxx command line parameter. For example to use the kvm2 driver with clusterctl you woud add --bootstrap-flags vm-driver=kvm2, for linux, if you haven't installed any driver, you can add --bootstrap-flags vm-driver=none
```

Comment:
s/--bootstrap-flags vm-driver=xxx/`--bootstrap-flags vm-driver=xxx`
s/--bootstrap-flags vm-driver=kvm2/`--bootstrap-flags vm-driver=kvm2`
s/--bootstrap-flags vm-driver=none/`--bootstrap-flags vm-driver=none`

Reply:
I think we should probably avoid talking about `vm-driver=none` since that will actually modify the host system.

Reply:
ok